plugins/yellow-devin/commands/devin/delegate.md

```
@@ -56,47 +67,85 @@ Branch: {branch_name}
+
+printf '%s' "$payload" | \
   curl -s --connect-timeout 5 --max-time 60 \
     -w "\n%{http_code}" \
-    -X POST "https://api.devin.ai/v1/sessions" \
-    -H "Authorization: Bearer $DEVIN_API_TOKEN" \
+    -X POST "${ORG_URL}/sessions" \
+    -H "Authorization: Bearer $DEVIN_SERVICE_USER_TOKEN" \
     -H "Content-Type: application/json" \
     -d @-
 ```
 
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)